packages/victory-box-plot/src/helper-methods.tsx

```diff
@@ -152,7 +152,7 @@ export const getDomain = (props, axis) => {
   return Domain.createDomainFunction(getDomainFromData)(props, axis);
```

Comment:
This file was converted to TS, but not modified a ton. So for an example like this:

```ts
const nanToNull = (val) => (isNaN(val) ? null : val);
```

that `nanToNull` function's argument `val` has a type of `any`, because we haven't declared a type for the arg. I'm wondering if we should be tightening down on this, but I'll defer to @scottrippey for what he thinks (not sure what we've been doing for previous TS conversions, and how strict we're being in internal/helper methods).

(Side note: the reason TS doesn't complain about this is because we have `"noImplicitAny": false,"` in our `tsconfig.base.json`. If we wanted to be a bit more strict in this regard, we'd flip that to `true` – but it's a pretty big commitment, especially in a codebase this large.)

Reply:
@gksander while @scottrippey and I were pairing on some of the `victory-axis` conversions, he'd had mentioned that we can just port over the helper files for now to and prioritize getting the entire codebase in TS first and then strictly typing those functions in the future when we come across them in the future.

Happy to change things up if you think there are more concerning functions that should be typed now though!

Reply:
That sounds fine to me, Scott's been leading the charge – so if it's good with him, it's good with me 😅